src/extensions/scratch3_speech/index.js

```diff
@@ -685,29 +686,51 @@ class Scratch3SpeechBlocks {
     getInfo () {
         return {
             id: 'speech',
-            name: 'Google Speech',
+            name: formatMessage({
+                id: 'speech.extensionName',
+                default: 'Google Speech',
+                description: 'Name of extension that adds speech recognition blocks.'
```

Comment:
We may want to add 'Do not translate Google' to the description. I'm guessing that people usually wouldn't, but it doesn't hurt to be explicit.

Reply:
Good point! Done! The conventions are all starting to come back to memory :)